yaksh/templates/yaksh/course_modules.html

```diff
@@ -15,22 +15,17 @@
 {% block main %}
 
 <div class="container">
-<div class="row justify-content-md-center yakshwell ">
+<div class="row justify-content-md-center yakshwell">
     <div class="col-md-10 bg-light card">
-        <div class="row align-items-center">
+        <div class="row align-items-center" style="height: 100px;">
             <div class="col h4">  {{ course.name }}  </div>
-            <div class="col-md-3 ml-auto yakshwell">
-                <div class="row align-items-center"> 
-                    <div class="col-sm-6"> Overall
-                    Course</br>Completion :
-                    </div>
-                    <div class="col-sm-5">
-                    <span class="progress">
-                      <span class="progress-bar bg-warning " role="progressbar" style="width: {{ course_percentage }}%; color: black;" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100">{{ course_percentage }} % completed</span>
-                    </span>
-                    </div>
-                </div>
-            </div>   
+            <div class = "col">
+                {% if course.has_lessons %}&nbsp;&nbsp;&nbsp;
```

Comment:
Avoid the use of so many```&nbsp;``` !
Use proper divs or HTML elements or CSS property instead.

Reply:
@prathamesh920 Will it be okay to create a new class selector just for one tag in css file?

Reply:
same as above.
Also, use proper divs or HTML elements or CSS property instead